srcpkgs/srb2/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'srb2'
+pkgname=srb2
+version=2.2.8
+revision=1
+wrksrc="SRB2-SRB2_release_${version}"
+create_wrksrc=true
+build_wrksrc=${wrksrc}
+build_style=gnu-makefile
+make_use_env=yes
+make_build_args=" -C src LINUX=1 USE_OPENMP=1 EXENAME=${pkgname}
+ DBGNAME=${pkgname}-debug NOOBJDUMP=1 NOUPX=1"
+hostmakedepends="pkg-config"
+makedepends="SDL2-devel SDL2_mixer-devel libpng-devel libupnp-devel libcurl-devel libgme-devel libopenmpt-devel"
+short_desc="3D Sonic fan game based off of Doom Legacy"
+maintainer="oreo639 <oreo6391@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://srb2.org/"
+distfiles="https://github.com/STJr/SRB2/archive/SRB2_release_${version}.tar.gz
+ https://github.com/STJr/SRB2/releases/download/SRB2_release_${version}/SRB2-v${version}-Full.zip"
+checksum="bbb2777a05a16019d8b4a3fcef9829e6e7cfd795710b3ffcce009120f0e2c43c
+ 7a3805b1127bd42db6432a9fe3e25927269e334497d48bdc9c2751edc114d718"
+
+case "$XBPS_TARGET_MACHINE" in
+	i686*) makedepends+=" nasm" ;;
+	x86_64*) make_build_args+=" X86_64=1 NONX86=1" ;;
+	*) make_build_args+=" NONX86=1" ;;
+esac
+
+if [ -n "$CROSS_BUILD" ]; then
+	make_build_args+=" OBJCOPY=${XBPS_CROSS_TRIPLET}-objcopy"
+fi
+
+do_install() {
+	vbin bin/Linux/Release/${pkgname}
+	vinstall srb2.png 644 /usr/share/pixmaps
+	vinstall ${FILESDIR}/srb2.desktop 644 /usr/share/applications/
```

Comment:
What about this file? They have a template in https://github.com/STJr/SRB2/blob/master/debian-template/srb2.desktop , you should be able to talk them into adding a pre-generated one with reasonable defaults. Or even generate it at build time.

Reply:
Sure.